assets/i18n/en_US.json

```
@@ -162,6 +162,12 @@
         "exportApkButtonTooltip": "Export patched APK",
         "exportLogButtonTooltip": "Export log",
 
+        "patchErrorDialogTitle": "Something went wrong",
+        "patchErrorDialogText": "During patching an exception occurred.",
```

```suggestion
        "patchErrorDialogText": "An error occured while patching.",
```

Maybe this would be better?